src/app.py

```diff
@@ -107,5 +107,27 @@ def add_course():
     return redirect(url_for("start"))
 
 
+# Router to course detailed page
+@app.route("/course_page", methods=["GET", "POST"])
+def course_page():
+    df = get_df_from_csv_in_s3(s3, bucket_name, mock_data_file)
+    courses = df.loc[0, "courses"]  # For PoC purpose
+    # Parsing it into a Python list
+    courses = ast.literal_eval(courses)
```

Comment:
`courses` is a global variable defined above so I think you could directly access it instead of retrieving all these data again from `s3`